index.html

```diff
@@ -15,8 +15,15 @@
     />
   </head>
   <body>
-    <div id="swquiz-app"></div>
-
+    <div id="swquiz-app">
+      <header>
+        <div class="header container">
+          <a href="./">
```

Comment:
Since you use logo image as background, there is no written information what is on the logo. Consider adding written explanation what does the link contain. Perhaps hidden, only available for screen readers.

Reply:
I think that the class header__logo descript enough what is inside div, but I can add some comment if its necessary

Reply:
Maybe something like "star wars logo"?